data/distro.yml

```diff
@@ -510,6 +510,40 @@
       <p>Now all you have to do is <a href="https://flathub.org/">install some apps</a>!</p>
     </ol>
 
+- name: "Trisuqel"
+  logo: "trisquel.svg"
+  info: >
+    <ol class="distrotut">
+      <li>
+        <h2>Install Flatpak</h2>
+        <p>To install Flatpak on Trisquel simply open the Terminal ( MATE Terminal ) and enter :</p>
+        <pre><code>
+          <span class="unselectable">$</span> sudo apt install flatpak
+        </code></pre>
+        <p>Then just press enter. APT may also ask you to confirm you want to install the application.</p>
+      </li>
+      <li>
+        <h2>Install the Software Flatpak plugin</h2>
+        <p>Flatpak does not currently have a plugin for MATE's Add/Remove Software Application but you can install Flatpak apps with a Graphic Interface using GNOME Software:</p>
```

Comment:
Is it recommended (officially supported) to install GNOME Software in Trisquel?

Reply:
> Is it recommended (officially supported) to install GNOME Software in Trisquel?

As I understand it Trisquel officially comes preinstalled with a modified of GNOME Install/Synaptic. The package name is `trisquel-app-install`. 

GNOME Software, is not currently supported by Trisquel based on a dispute over its dependence on Snapd. [[1](https://trisquel.info/en/issues/23466)]. 

KDE Discover, is a program I have yet to fully test issue trackers and forum posts exist for it as well. [[1](https://trisquel.info/en/issues/28357)] [[2](https://trisquel.info/en/forum/should-discover-be-bundles-triskel)]

_______________________________________________________

Despite not being officially supported, having a GUI tool to install apps can be really good for people who are a bit newer to the GNU/Linux ecosystem. However, I do understand if you wish to only list how to add plugins for officially supported package managers as well. :+1:

Reply:
I would like to add an update, Triskel is a version of Trisquel with KDE but it doesn't appear to have official support for Discover. [[4](https://trisquel.info/en/forum/triskel-9-updater)]

Reply:
Any chance this PR could be updated to reflect that changes?

Reply:
Ping.

Reply:
(You will also need to rewrite this PR using a new metadata format that the yaml file uses. See how other entries do look like now: https://github.com/flatpak/flatpak.github.io/blob/source/data/distro.yml#L1)